xdc-plugin/xdc.cc

```diff
@@ -44,22 +44,23 @@ static bool isOutputPort(RTLIL::Wire* wire) {
 	return wire->port_output;
 }
 
-enum class SetPropertyOptions { INTERNAL_VREF, IOSTANDARD, SLEW, DRIVE, IN_TERM };
+enum class SetPropertyOptions { INTERNAL_VREF, IOSTANDARD, SLEW, DRIVE, IN_TERM, LOC };
```

Comment:
Please rename LOC to whatever the new parameter is going to be named

Reply:
Changed to `IO_LOC_PAIRS` as this is what it is.